protocols/messages.go

```diff
@@ -211,13 +211,8 @@ func SummarizeMessage(m Message) MessageSummary {
 
 // CreateSignedProposalMessage returns a signed proposal message addressed to the counterparty in the given ledger
 // It contains the provided signed proposals and any proposals in the proposal queue.
-func CreateSignedProposalMessage(ledger *consensus_channel.ConsensusChannel, sp ...consensus_channel.SignedProposal) Message {
-	recipient := ledger.Leader()
-	if ledger.IsLeader() {
-		recipient = ledger.Follower()
-	}
-	// Append the proposals to any existing proposals in the queue
-	proposals := append(ledger.ProposalQueue(), sp...)
+func CreateSignedProposalMessage(recipient types.Address, proposals ...consensus_channel.SignedProposal) Message {
```

Comment:
I wonder if we could improve some of the function names. I don't have a great suggestion, but itt seems awkward that the follower creates a signed "proposal" message. It sounds like they are making a proposal, which they aren't.

Is the follower sending a "countersigned proposal"? 

Are both roles merely sending a LedgerProtocolMessage?

Reply:
>I wonder if we could improve some of the function names. I don't have a great suggestion, but it seems awkward that the follower creates a signed "proposal" message. It sounds like they are making a proposal, which they aren't.

I think the problem is that a `SignedProposal` is overloaded. We used a `SignedProposal` in a message to either:
- send a new proposal from the leader to the follower
- accept a proposal as the follower by sending out a `SignedProposal` with the follower's signature.

Maybe we should look at renaming `SignedProposal` in the message so it's less confusing?